scipy-1.0/paper.tex

```diff
@@ -920,27 +920,17 @@ \subsection*{Numerical optimization}
 
 \subsection*{Statistical distributions}
 
-The \texttt{scipy.stats} module provides more than 100 probability
-distributions and a framework to implement additional distributions. Given an
-expression for a probability distribution function (pdf), the framework will
-calculate moments, cumulative distribution function (cdf), differential
-entropy, and more. Those properties are calculated based on generic code, e.g.,
-numerical integration of the pdf to obtain the cdf.  Where possible, the
-generic code is replaced by explicit formulas or more accurate or efficient
-implementations. For example, most distributions have an explicit formula for
-the cdf, so it is used instead of integrating the pdf. The distributions
-framework also allows the user to draw random variates from the distribution.
-Drawing random variates now accepts a \texttt{random\_state} parameter, which
-is either a \texttt{numpy.random.RandomState} object or an integer used to
-generate such an object, to provide fine-grained user-level control over the
-random state of a workflow.  Currently about 80 continuous univariate
-distributions, 12 discrete univariate distributions and 10 multivariate
-distributions are implemented.  Key recent distributions added to
-\texttt{scipy.stats} include the histogram-based distribution in
-\texttt{scipy.stats.rv\_histogram} and the multinomial distribution in
-\texttt{scipy.stats.multinomial} (used, for example, in natural language
-processing\cite{Griffiths5228}). \texttt{multinomial} is the first and so far
-the only multivariate discrete distribution available in SciPy.
```

Comment:
I think the issue with dropping all of this is that the whole section is about recent improvements, and now there's only the generic description left. The description bit is nicer now, but I would suggest to at least keep `multinomial`.

Reply:
agree, I added back that part